metrics/.env

```diff
@@ -0,0 +1 @@
+SLACK_WEBHOOK_URL=https://hooks.slack.com/
```

Comment:
PR seems good, just this one line - what uses it?

Reply:
ah somehow trickled in from branch above - removed it - 6627ce33